helm/upp-aggregate-healthcheck/templates/ingress.yaml

```diff
@@ -1,41 +0,0 @@
-{{ if eq .Values.ingress.enabled "true" }}
```

Comment:
There is another nginx ingress file in the repo `operation-ingress.yaml` that needs to be removed. Besides that should be use `PATH_PREFIX` instead of plain `/__health`?